great_expectations/expectations/metrics/map_metric_provider.py

```diff
@@ -565,6 +727,83 @@ def _pandas_column_map_condition_values(
         return list(domain_values[: result_format["partial_unexpected_count"]])
 
 
+def _pandas_multicolumn_map_condition_values(
+    cls,
+    execution_engine: PandasExecutionEngine,
+    metric_domain_kwargs: Dict,
+    metric_value_kwargs: Dict,
+    metrics: Dict[str, Any],
+    **kwargs,
+):
+    """Return values from the specified domain that match the map-style metric in the metrics dictionary."""
+    (
+        boolean_mapped_unexpected_values,
+        compute_domain_kwargs,
+        accessor_domain_kwargs,
+    ) = metrics["unexpected_condition"]
+    df, _, _ = execution_engine.get_compute_domain(
+        domain_kwargs=compute_domain_kwargs,
+        domain_type=MetricDomainTypes.IDENTITY.value,
```

Comment:
nit: this seems inconsistent...why are we using the (string) value here? Let's stick to the enum value.

```suggestion
        domain_type=MetricDomainTypes.IDENTITY,
```

Reply:
@jcampbell Fixed.  Good catch.  It was a typo, since I already fixed it everywhere else to use the enum.  Thank you.